lib/Model/UserData.php

```diff
@@ -279,11 +271,10 @@ public function __construct(array $data = null)
         $this->container['private_custom_data'] = $data['private_custom_data'] ?? null;
         $this->container['created_date'] = $data['created_date'] ?? null;
         $this->container['last_seen_date'] = $data['last_seen_date'] ?? null;
-        $this->container['platform'] = $data['platform'] ?? null;
-        $this->container['platform_version'] = $data['platform_version'] ?? null;
-        $this->container['device_model'] = $data['device_model'] ?? null;
-        $this->container['sdk_type'] = $data['sdk_type'] ?? null;
-        $this->container['sdk_version'] = $data['sdk_version'] ?? null;
+        $this->container['platform'] = 'PHP';
+        $this->container['platform_version'] = PHP_VERSION;
+        $this->container['sdk_type'] = 'server';
+        $this->container['sdk_version'] =  '1.0.6';
```

Comment:
Do we have to manually update sdk_version whenever we release a new version? would be cool if there was some way to fill this automatically

Reply:
Was going to suggest that if it has to be manual that the release docs be updated to include that fact, but I see you already did that https://www.notion.so/taplytics/How-to-release-server-SDKs-to-package-managers-9465d668b5d641e5990f4bbfe4c6744b#8120e0d0fafb4c48861c22cabee36cdc  good job!

Reply:
Doesn't appear to be an automatic way, will have to be a manual step as part of the release process every time